examples/generic-plotting.py

```diff
@@ -0,0 +1,142 @@
+
+#%%
+import resdx
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import seaborn as sns
+sns.set()
+
+import numpy as np
+#%% Plotting function class
+def makeplots(ConditionsArray,FunctionsArray,FunctionsArray_2nd_axis = None):
```

Comment:
It is important that there are example files illustrating the capability, but this functionality still needs to a method of the DXUnit class and not a free-floating function in the example file.